estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class EstatePropertyTag(models.Model):
+    _name = "estate.property.tag"
+    _description = "Estate Property tag"
+    _order = "name"
+
+    name = fields.Char(required=True)
+    color = fields.Integer()
+
+    _sql_constraints = [
```

Comment:
all the private attributes belong together, so the `_sql_constraints` should go right after `_description` and `_order`